### PR TITLE
don't create the cni temporal config file in the cni configuration folder

### PR DESCRIPTION
**Pull request by aojea**
Fixes: https://github.com/aojea/kindnet/issues/42

### CODE REVIEW

File: cmd/kindnetd/cni.go [hunk 1]
```diff
@@ -172,10 +172,8 @@ func (c *CNIConfigWriter) Write(inputs CNIConfigInputs) error {
 		return nil
 	}
 
-	// use an extension not recognized by CNI to write the contents initially
-	// https://github.com/containerd/go-cni/blob/891c2a41e18144b2d7921f971d6c9789a68046b2/opts.go#L170
-	// then we can rename to atomically make the file appear
-	f, err := os.Create(c.path + ".temp")
+	// use a temporal file so we can rename to atomically make the file appear
```
Comment (dims) [inline cmd/kindnetd/cni.go:175]: typo - "temporary"

**Comment by dims** [comment]
tiny nit, otherwise LGTM

**Comment by aojea** [comment]
does not work :( https://github.com/aojea/kindnet/issues/42#issuecomment-2241157681

**Pull request closed without merging**